### ISSUE TITLE
Windows: UTF-8 input doesn't quite work

### ISSUE THREAD

**Issue by cyanite** [gwsw/less#37]
When building less on Windows (with Visual C++) utf-8 input (when the codepage for the current console is set to 65001), doesn't work correctly.

When I press / to enter search and then enter an "æ", for instance, it doesn't display anything and freezes up or starts flashing a lot and doesn't respond after that. On an older machine it doesn't do this, but instead turns the æ into "^@'" (and ø and å into "^@(" and "^@^Z" respectively).

The version shipped with git, for example, doesn't have this problem, but is also not built against Visual C++, but rather MSYS2.

**Comment by zeromake** [comment]
@cyanite 
我现在重新试了一下发现不会卡住，但是无法正常输入 æ。
尝试在 main 函数开头添加
``` c
#ifdef _WIN32
	SetConsoleCP(65001);
	SetConsoleOutputCP(65001);
#endif
```
发现输入正常了。

**Comment by gwsw** [comment]
Seems to be fixed in 03f4fb1e1711d41adbb301fdbcf5963cc225856f by the fixes for #391.
